bastion/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.9'
```

Comment:
you don't need this anymore. The docker runtime can decode this on its own in the new version of the engine.

![IMG_9634](https://github.com/bifrostlab/deploy-llm-bot/assets/4188758/22b7956f-8724-4353-ad1a-9f0af5e77129)